crates/core/Cargo.toml

```diff
@@ -29,7 +29,9 @@ derive_more.workspace = true
 dunce = "1.0.5"
 home = "0.5.9"
 ignore = { version = "0.4.22", features = ["simd-accel"] }
+futures = "0.3.30"
```

Comment:
Was this used anywhere?

Reply:
Mb, it was leftover from an experiment haha, I tried converting ReadDir to tokio_stream::wrappers::ReadDirStream to see if that would improve the async version of find_install_path. I did make some changes since the new commit, but it doesn't involve the futures crate